src/CalculatorUnitTests/CurrencyConverterUnitTests.cpp

```diff
@@ -407,11 +407,11 @@ namespace CalculatorUnitTests
             const UCM::Unit usdUnit = GetUnit(unitList, L"USD");
             const UCM::Unit eurUnit = GetUnit(unitList, L"EUR");
 
-            VERIFY_ARE_EQUAL(StringReference(L"United States - Dollar"), ref new String(usdUnit.name.c_str()));
-            VERIFY_ARE_EQUAL(StringReference(L"USD"), ref new String(usdUnit.abbreviation.c_str()));
+            VERIFY_ARE_EQUAL(wstring(L"United States - Dollar"), usdUnit.name);
```

Comment:
Thanks for fixing these up!  For future reference, you can use [operator""s](https://en.cppreference.com/w/cpp/string/basic_string/operator%22%22s) to cleanly write a string literal that is converted to a `wstring`.  For example, this is equivalent `L"United States - Dollar"s`.

Reply:
The VERIFY_ARE_EQUAL macro doesn't specify a type for the param, so wstring needs to be specified in order for the == operator to succeed.